lib/util/news.ts

```diff
@@ -0,0 +1,43 @@
+import type { Languages } from '../hooks/useLanguage'
+import { languages } from '../hooks/useLanguage'
+import { z } from 'zod'
+
+export type News = {
+  title: string,
+  date: Date,
+  description: (string | URL)[],
+  externalResource?: URL,
+  keys: string[]
+}
+
+export type LocalizedNews = Record<Languages, News[]>
+
+export const newsSchema = z.object({
+  title: z.string(),
+  description: z.string(),
+  date: z.string(),
```

Comment:
looking at http://cdn.helpwave.de/feed_intl.json shows this to be a string such as `"10 July 2023"`, I'd suggest sticking to ISO 8601 even though you might end up with some timezone trickery. Do note that currently your dates are not localized per language but all in english.

You can use the Intl api for displaying the dates: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat